docs/docs/multi-core-builds.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Multi-core builds
+---
+
+Gatsby now performs the static HTML generation phase of the overall [Page HTML Generation](/docs/html-generation/) process using multi-core parallel pools of workers. This helps speed up builds by distributing build generation tasks across multiple cores on your machine.
+
+By default Gatsby creates a pool of workers equal to the number of physical cores on your machine, see [build-html.js](/docs/html-generation/#build-htmljs)
```

Comment:
Punctuation tip: Introductory phrases need commas after them in order to mimic natural speech. For example
> By default, Gatsby...
> In some scenarios, it may...

Commas around nonessential / parenthetical phrases help to mimic the way people speak. Example:
> **For example**, if you are running a Cloud server (like AWS EC2), your DevOps engineers may

Would you please update commas in this doc?

Reply:
Updated :)